src/fides/ctl/core/config/redis_settings.py

```diff
@@ -14,7 +14,7 @@ class RedisSettings(FidesSettings):
     host: str = "redis"
     port: int = 6379
     user: Optional[str] = ""
-    password: str = "testpassword"
+    password: str
```

Comment:
```suggestion
    password: str = input("Field required: redis.password\n")
```

Something similar to this feels like a much better user experience. We shouldn't require something but then not help the user triage the issue